apps/precursor-chemicals/views/upload-company-certificate.html

```diff
@@ -0,0 +1,52 @@
+{{<partials-page}} {{$encoding}}enctype="multipart/form-data" name="file-upload-form" {{/encoding}} {{$page-content}}
+
+  <div class="govuk-inset-text">
+    {{#markdown}}file-requirements{{/markdown}}
+  </div>
+
+  <div class="govuk-form-group govuk-!-margin-top-6" id="hofFileUpload">
+    <label class="govuk-label govuk-label--m" for="file-upload">
+        {{#t}}fields.upload-company-certificate.label{{/t}}
+    </label>
+    <p id="file-upload-error-maxFileSize" class="govuk-error-message govuk-!-display-none">
+      <span id="validation-error" class="govuk-visually-hidden">{{#t}}journey.error{{/t}}:</span> {{#t}}validation.upload-company-certificate.maxFileSize{{/t}}
+    </p>
+    <p id="file-upload-error-fileType" class="govuk-error-message govuk-!-display-none">
+      <span id="validation-error" class="govuk-visually-hidden">{{#t}}journey.error{{/t}}:</span> {{#t}}validation.upload-company-certificate.fileType{{/t}}
+    </p>
+    <input class="govuk-file-upload" id="file-upload" name="file-upload" type="file" value="company-registration-certificate">
+    <div id="upload-page-loading-spinner" class="spinner-container">
+      <div class="spinner-loader"></div>
+      <span class="spinner-message">{{#t}}journey.uploading{{/t}}</span>
+    </div>
+  </div>
+
+  {{^values.company-registration-certificate}}
+  <h2 class="govuk-heading-s">No documents uploaded</h2>
+  {{/values.company-registration-certificate}}
+
+  {{#values.company-registration-certificate.length}}
+  <h2 class="govuk-heading-s">You have uploaded {{values.company-registration-certificate.length}} file(s)</h2>
```

Comment:
Think this heading should be 'Files uploaded'

Reply:
Also, it's worth moving this to translations